dev/com.ibm.ws.microprofile.faulttolerance_git_fat_tck/build.gradle

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+task clone {
+  mustRunAfter cleanFat
+  doLast{
+    gitClone "microprofile-fault-tolerance"
```

Comment:
if we are going to do this, I'd prefer to keep the logic in this FATs build.gradle file, instead of cluttering the common FAT gradle script with something a small number of FATs are requiring

Reply:
I think there will be a few fat.gradles that call this  - I am just about to create some more - for config so I don't want to have copy and paste the same code multiple times. If I deliver version 1.0 then it can be factored later but I really know I want to keep the build.gradles small as there will be about half a dozen of them perhaps.